src/streams.h

```diff
@@ -430,9 +430,18 @@ class AutoFile
     /** Implementation detail, only used internally. */
     std::size_t detail_fread(Span<std::byte> dst);
 
+    /** Wrapper around fseek(). Will throw if seeking is not possible. */
     void seek(int64_t offset, int origin);
+
+    /** Find position within the file. Will throw if unknown. */
     int64_t tell();
 
+    /** Wrapper around FileCommit(). */
+    bool Commit();
+
+    /** Wrapper around TruncateFile(). */
```

Comment:
isn't this obvious from the implementation?

Reply:
I'm not exactly sure in which cases this comment would help us avoid having to check the sources - but I don't have strong opinions about it, please resolve this comment.